presto-hive/src/main/java/io/prestosql/plugin/hive/HiveClientConfig.java

```diff
@@ -800,20 +801,6 @@ public HiveClientConfig setOrcDefaultBloomFilterFpp(double orcDefaultBloomFilter
         return this;
     }
 
-    @Deprecated
-    public boolean isOrcOptimizedWriterEnabled()
-    {
-        return orcOptimizedWriterEnabled;
-    }
-
-    @Deprecated
-    @Config("hive.orc.optimized-writer.enabled")
```

Comment:
When removing old writer, we need a new toggle, that would make the new writer "fake" writer version (and writer id).